File: 백준/Bronze/2577. 숫자의 개수/숫자의 개수.java
import java.util.*;
import java.io.*;

public class Main {

    public static void main(String[] args) throws IOException {

        Scanner sc = new Scanner(System.in);

        int result = 1;
        // A * B * C
        for(int i = 0; i < 3; i++) {
            result *= sc.nextInt();
        }
        // 결과 문자열화
        String temp = String.valueOf(result);

        // 0부터 1까지 몇번 쓰였는지 체크
        for(int i = 0; i < 10; i++) {
            int n = i;
            int numberCount = (int)temp.chars()
                    .filter(num -> Integer.parseInt(String.valueOf((char)num)) == n)
                    .count();
            System.out.println(numberCount);
        }
    }
}

File: 백준/Bronze/2577. 숫자의 개수/숫자의 개수.kt
import java.util.*

fun main() = with(Scanner(System.`in`)) {

    var temp = 1

    repeat(3) {
        temp = temp.times(nextInt())
    }

    val str : String = temp.toString()

    for(i : Int in 0..9)
        println(str.count { it.toString() == i.toString() })
}